sdk/communication/communication-network-traversal/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0 (2022-02-03)
+## 1.0.0 (2022-02-04)
```

Comment:
wait - we shouldn't be doing this - @sadasant

Reply:
my git history got wrong in this PR. I’ll fix it